docs/02-get-started/05-observability.md

```diff
@@ -20,16 +20,40 @@ To do that, we'll use the Grafana dashboard that comes with Kyma.
 
 ## View the logs
 
-1. In the Grafana dashboard, use the left menu to navigate to **Explore** and choose **Loki** from the dropdown list.
+<div tabs name="View the logs" group="view-logs">
+  <details open>
+  <summary label="Grafana dasboard">
+  Grafana dashboard
+  </summary>
+
+1. Use the left menu to navigate to **Explore** and choose **Loki** from the dropdown list.
 2. Click on **Log browser** and select the following values:
    - **1. Select labels to search in**: `container`, `function`
-   - **2. Find values for the selected labels**: for **function** choose `lastorder`, for **container** choose `function` 
-   
-   Click **Show logs**.
+   - **2. Find values for the selected labels**: for **function** choose `lastorder`, for **container** choose `function`
+  
+3. Click **Show logs**.
 
     > **NOTE:** Alternatively, type or paste the `{function="lastorder", container="function"}` query and press `Shift`+`Enter` or click on **Run query**. 
 
-You can now browse the logs. 
+You can now browse the logs.
+
+  </details>
+  <details>
+  <summary label="Kyma Dashboard">
+  Kyma Dashboard
+  </summary>
+
+1. Go to the `default` Namespace.
+   
+2. Go to **Workloads** > **Functions**.
+   
+3. Select your `lastorder` Function.
+   
+4. Scroll down and click on **View Logs**.
+
+You can now browse the logs.
+  </details>
+</div>
```

Comment:
2) The whole thing looks good in GitHub when just this bit is displayed, but when you open the [Website preview](https://deploy-preview-13068--kyma-project-docs-preview.netlify.app/docs/kyma/preview/02-get-started/05-observability/) and look at the whole document, it doesn't make sense to first Access Grafana only to later view the logs in Kyma Dashboard (the user doesn't know it yet, if they go through the sections one by one, that they don't have to use Grafana for this). 
I suggest that you either remove the toggles, and just add a **NOTE** right under the title of the **View the logs** section that you can also do it in Kyma Dashboard: 

```
**NOTE:** You can also view the logs in Kyma Dashboard itself. Go to the `default` Namespace > **Workloads** > **Functions** > your `lastorder` Function, scroll down and click on **View Logs**.  
```

or add a toggle between Grafana/Kyma Dashboard for the whole document, which is less desirable, since you then loose the right-side navigation + you can't view the metrics in Kyma Dashboard (right?), so I'd advise against it. 

![image](https://user-images.githubusercontent.com/53601578/149501153-9b5cc168-58d1-4c98-b89e-57b421bc157d.png)